### PR TITLE
UI: Adjustments to YouTube integration strings

### PR DESCRIPTION
**Pull request by Warchamp7**
### Description
Changes some of the English strings around YouTube integration. Various tweaks to phrasing, casing, and consistency with other places in the UI.

### Motivation and Context
We don't use all caps anywhere ever, so I removed that.

Updated the titles of the tabs to be capital case, same as the Output tab.

Fixed mixed use of "auto start" and "auto-start"

Other phrasing tweaks to match the verbiage of the rest of the tabs (Ex. Select event vs choose event)

### How Has This Been Tested?
Checked all updated strings still displayed properly throughout the UI

### Types of changes
- New feature (non-breaking change which adds functionality)

### Checklist:
- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

**Review by WizardCM — APPROVED** [review]

File: UI/data/locale/en-US.ini [hunk 1]
```diff
@@ -1240,7 +1240,7 @@ YouTube.Actions.Notify.Title="YouTube"
 YouTube.Actions.Notify.CreatingBroadcast="Creating a new Live Broadcast, please wait..."
 
 YouTube.Actions.AutoStartStreamingWarning.Title="Manual start required"
-YouTube.Actions.AutoStartStreamingWarning="Auto start is disabled for this stream, click the \"GO LIVE\" button to publish your broadcast on YouTube."
+YouTube.Actions.AutoStartStreamingWarning="Auto-start is disabled for this event, click \"Go Live\". to start your broadcast"
```
Comment (Vainock) [inline UI/data/locale/en-US.ini:1243]: The period should be moved to the end of the string.

**Review by Vainock — CHANGES_REQUESTED** [review]

File: UI/data/locale/en-US.ini (hunk 1 above)
Comment (Warchamp7) [reply UI/data/locale/en-US.ini:1243]: Good catch

**Comment by derrod** [comment]
I believe squash + merge is in order, any disagreements?

**Comment by Warchamp7** [comment]
Squashed

**Pull request merged**